src/third/redux-persist/createPersistor.js

```diff
@@ -66,8 +68,12 @@ export default function createPersistor (store, config) {
           return
         }
 
-        let [storageKey, serializedValue] = storesToProcess.shift()
-        storage.setItem(storageKey, serializedValue, warnIfSetError(storageKey))
+        let updatesFromQueue = storesToProcess.shift()
+        // Warning: not guaranteed to be done in a transaction.
+        storage.multiSet(
+          updatesFromQueue,
+          warnIfSetErrors(updatesFromQueue.map(([storageKey, serializedValue]) => storageKey).join(', '))
```

Comment:
How about we pass the `updatesFromQueue.map(([storageKey, serializedValue]) => storageKey)` in to `warnIfSetErrors`, and then do the formatting in there, rather than `join`ing here?